README.md

```diff
@@ -201,6 +201,16 @@ message = client.messages.create(
 print(message)
 ```
 
+The bedrock client supports the following arguments for authentication
+
+```py
+AnthropicBedrock(
+  aws_profile='...',
+  aws_region='us-east'
+  ... etc
```

Comment:
ah sorry I wasn't clear in my comment, I intended for `... etc` to be replaced with an exhaustive list of the supported options
```suggestion
  aws_secret_key='...',
  aws_access_key='...',
  aws_session_token='...',
```

Reply:
Ahh fine, sorry, i did it.